bundles/statistics/statsgrid2016/service/ClassificationService.js

```diff
@@ -358,7 +361,7 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.ClassificationService',
 
                 textSvgEl.find('text').attr({
                     x: valPos.x2 + 4,
-                    y: valPos.y2 + fontSize/2
+                    y: valPos.y2 + classificationLabelCorrection(index)
```

Comment:
SVG `<text>` elements have `dy` attribute that we can use for adjusting the height of the text relative to some "normal" position.

Reply:
Elements are not nested here, so the "normal" is 0,0. Using dx & dy would be a better solution, but requires more refactoring so we are leaving it out of the scope by now.

Reply:
Ok.